contracts/bond/Roles.sol

```diff
@@ -2,16 +2,21 @@
 pragma solidity ^0.8.0;
 
 /**
- * @title Roles within the Bond access control schema.
+ * @title Roles within the hierarchical DAO access control schema.
  *
- * @notice The DAO grants all other roles (role admin for all roles), while not permitted any actions itself.
+ * @notice Similar to a Linux permission system there is a super user, with some of the other roles being tiered
+ *          amongst each other.
  *
- *  BOND_ADMIN role is deals with the Bond life cycle.
- *  BOND_AGGREGATOR roles collates bonds together in one place.
- *  SYSTEM_ADMIN role deals with upgrading the contract.
+ *  SUPER_USER role the manage for DAO_CREATOR roles, in addition to being a super set to to all other roles functions.
+ *  DAO_CREATOR role only business is creating DAOs and their configurations.
+ *  DAO_ADMIN role can update the DAOs configuration and may intervene to sweep / flush.
+ *  DAO_MEEPLE role is deals with the life cycle of the DAOs products.
+ *  SYSTEM_ADMIN role deals with tasks such as pause-ability and the upgrading of contract.
  */
 library Roles {
+    bytes32 public constant SUPER_USER = "SUPER_USER";
     bytes32 public constant DAO_ADMIN = "DAO_ADMIN";
-    bytes32 public constant BOND_ADMIN = "BOND_ADMIN";
+    bytes32 public constant DAO_CREATOR = "DAO_CREATOR";
```

Comment:
How are DAO_CREATOR different to DAO_ADMIN?

Reply:
`DAO_CREATOR` and `SUPER_USER` are the only roles that will be able to create a DAO via `createDao()`